src/lsp/cobol_config/cobol_config.ml

```diff
@@ -52,14 +52,18 @@ let default_search_path =
       match Sys.getenv_opt "XDG_CONFIG_HOME" with
       | Some p -> Some p
       | None -> Option.map (fun d -> d // ".config") (Sys.getenv_opt "HOME")
-    in
-    let system_specific =
+    and windose_specific =
```

Comment:
You do realize it's spelled "Windows", right ? ;)

Reply:
Yes. I'll let you fix that later if that really makes you sad :stuck_out_tongue_winking_eye:

Reply:
I'll survive ;)